deepmd/nvnmd/entrypoints/mapt.py

```diff
@@ -0,0 +1,321 @@
+
+import numpy as np
+import logging
+
+from deepmd.env import tf
+from deepmd.utils.sess import run_sess
+
+from deepmd.nvnmd.utils.fio import FioDic
+from deepmd.nvnmd.utils.config import nvnmd_cfg
+from deepmd.nvnmd.utils.weight import get_normalize, get_rng_s, get_filter_weight
+from deepmd.nvnmd.utils.network import get_sess
+
+from deepmd.nvnmd.data.data import jdata_deepmd_input
+
+from typing import List, Optional
+
+log = logging.getLogger(__name__)
+
+
+class MapTable:
+    r""" Generate the mapping table describing the relastionship of
+    r_ij->s_ij, r_ij->sr_ij, and s_ij->G_ij
+    
+    r_ij = |r_i - r_j|
```

Comment:
For the way to write math equations, maybe you can take a look at:

- https://deepmodeling--1707.org.readthedocs.build/projects/deepmd/en/1707/api_py/deepmd.descriptor.html#deepmd.descriptor.se_a.DescrptSeA
- https://deepmodeling--1707.org.readthedocs.build/projects/deepmd/en/1707/_modules/deepmd/descriptor/se_a.html#DescrptSeA